src/components/operational-review/hero.tsx

```diff
@@ -1,9 +1,18 @@
 import React from "react";
 import HeroBackgroundImage from "./hero-background-image";
-import { filteredHeroStyles } from '../common/filtered-hero-styles';
+import { filteredHeroStyles } from "../common/filtered-hero-styles";
+import { css } from "@emotion/core";
+
+const customCss = css`
```

Comment:
let's remove this css transform and just make sure the image is correct in the first place, would avoid some potential confusion if someone swaps the image for a different one